MTGSDKSwift/MTGAPIManager.swift

```diff
@@ -102,15 +46,10 @@ final class NetworkOperation {
         self.url = url
     }
     
-    func queryURL(completion: @escaping JSONCompletionWithError) {
-        
-        if Magic.enableLogging {
-            print("NetworkOperation - beginning queryURL... \n")
-        }
-        
+    func performOperation(completion: @escaping JSONCompletionWithError) {
         var networkError: NetworkError? {
             didSet {
-                completion(nil, networkError)
+                completion(Result.error(networkError!))
```

Comment:
I'd like to avoid all instances of force unwrap.  If at any time we assign `nil` to the networkError it would cause a crash.  Can we use a guard or a sensible default error instead?

Reply:
i will use guard and return. if it is being set to nil it doesnt make sense to call the completion anyway. Honestly the way I handled this in general is kinda odd. it works but I dont think its the optimal solution